src/scenes/MembershipMintPage/PartnerMembershipMintPage.test.tsx

```diff
@@ -0,0 +1,19 @@
+import { OpenseaAsset, reduceOpenseaAssetsToContractAddresses } from './PartnerMembershipMintPage';
+
+describe('reduceOpenseaAssetsToContractAddresses', () => {
+  test('', () => {
```

Comment:
Love that this is tested! This test block should have a name, currently its empty string

Reply:
added description